docs/src/_templates/custom_sidebar_logo_version.html

```diff
@@ -1,20 +1,21 @@
 {% if on_rtd %}
     {% if rtd_version == 'latest' %}
         <span title="latest development version">
-            <img src="https://img.shields.io/badge/latest_version-{{ version }}-orange?style=flat">
+            <img src="https://img.shields.io/badge/⚠️_version_('latest')-{{ version }}-gold?style=flat">
         </span>
     {% elif rtd_version == 'stable' %}
         <span title="stable version">
-            <img src="https://img.shields.io/badge/stable_version-{{ version }}-green?style=flat">
+            <img src="https://img.shields.io/badge/✨️_version_('stable')-{{ version }}-green?style=flat">
         </span>
     {% else %}
-        <span title="old version (not stable or latest)">
-            <img src="https://img.shields.io/badge/old_version-{{ version }}-red?style=?style=flat">
+        {# Covers builds for any specific versions/commits, including RC's. #}
+        <span title="archived version (not 'stable' or 'latest')">
+            <img src="https://img.shields.io/badge/🔒_version_(archived)-{{ version }}-red?style=?style=flat">
         </span>
     {% endif %}
 {%- else %}
     {# not on rtd #}
     <span title="development version">
-        <img src="https://img.shields.io/badge/dev_version-{{ version }}-blue?style=?style=flat">
+        <img src="https://img.shields.io/badge/✏️_version_(dev)-{{ version }}-blue?style=?style=flat">
```

Comment:
Regarding the sample "build without a version tag"
(from [comment]),
I don't understand why that is giving "version (archived) 5.7.0.dev1"
- I think saying "archived" it makes it look like a definite endpoint (like a post-release) instead of a moving goalpost -- e.g. what would 5.7.0.dev2 then be?

So. I was expecting that to come here + be labelled "version(dev)"
But I expect I misunderstood something ...

Reply:
When building via Read the Docs:

- `latest` = a dynamic label referring to whatever is the latest commit on `main`.
- `stable` = a dynamic label referring to the latest non-RC semantic-versioned tag.
- Any RTD build is static. Not only does this include all semantic-versioned tags (i.e. releases), but also any other tag/branch that has been selected for an RTD build.

Since these static builds never get rebuilt, most of them will be out of date. But this isn't consistently true - the most recent non-RC semantic-versioned build will correspond with `stable`; and the only way to access release candidate documentation is via its static build. So we want to make the reader aware that they COULD be looking at out of date documentation, or that it might be up to date right now but will go out of date in future. `archived` seemed like the best word (I also considered `fixed` but that seemed open to confusion about bug fixes).

Reply:
Note that we don't get an opportunity to re-label any static builds when they become out of date in future. We need a badge that will be appropriate now and in the future.

Reply:
Here's the outcome of further conversations with @pp-mo: 16c899e2249ab4b492fbb1d12a13467425eb9d23

See the main description for examples of the badges.